src/addons/dragAndDrop/compose.js

```diff
@@ -0,0 +1,14 @@
+// Exact copy over from recompose library
+// https://github.com/acdlite/recompose/blob/master/src/packages/recompose/compose.js
+
+export default function compose(...funcs) {
```

Comment:
if this is only used once can we just compose by hand and avoid vendoring this?

Reply:
Yeah, sounds good!